docs/.vuepress/config.js

```diff
@@ -26,14 +26,17 @@ module.exports = {
         type: 'text/css',
         href: 'https://cdn.jsdelivr.net/npm/keyrune@latest/css/keyrune.css',
       },
-    ]
+    ],
   ],
   // Vuepress Themeing
   themeConfig: {
+    algolia: {
+      apiKey: 'bfa11399ece64fbd87cbca3817bc498b',
```

Comment:
Should this be checked into the repo?

Reply:
Yes, it is a non-absusable search only key that is required to be public